components/partials/activity/positions/positions.vue

```diff
@@ -215,7 +214,6 @@ export default Vue.extend({
     pollOrderbooks() {
```

Comment:
Can we change the naming of the function? It should be `pollMarkPrice` or something like that.